zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -508,7 +509,11 @@ protected boolean jobAbort() {
     if (job != null) {
       job.setStatus(Status.ABORT);
     } else {
-      repl.cancel(getInterpreterContextWithoutRunner(null));
+      try {
+        repl.cancel(getInterpreterContextWithoutRunner(null));
+      } catch (InterpreterException e) {
+        throw new RuntimeException(e);
```

Comment:
why rewrap the exception here?

Reply:
Because InterpreterException is checked exception. If I throw it, I have to change signature of the caller method, this would cause lots of code change. In order to keep the PR small, I just throw it as RuntimeException.